website/docs/pype2/admin_presets_plugins.md

```diff
@@ -303,6 +304,7 @@ If source representation has suffix **"h264"** and burnin suffix is **"client"**
   | resolution_height | Resolution height. |
   | fps | Fps of an output. |
   | timecode | Timecode by frame start and fps. |
+  | focalLength | **Only available in Maya**<br /><br />Camera focal length per frame. Use syntax `{focalLength:.2f}` for decimal truncating. Eg. `35.234985` with `{focalLength:.2f}` would produce `35.23`, whereas `{focalLength:.0f}` would produce `35`. |
```

Comment:
Does this mean we'll need to set up a separate burnin profile just for maya?
Should the default settings start providing that?

Reply:
That might be a nice way of `introducing` people to the feature. Will have a look.